cpp/benchmarks/sort/sort.cpp

```diff
@@ -17,27 +17,30 @@
 #include <benchmarks/common/generate_input.hpp>
 
 #include <cudf/sorting.hpp>
+#include <cudf/table/table.hpp>
+#include <cudf/types.hpp>
 #include <cudf/utilities/default_stream.hpp>
 
 #include <nvbench/nvbench.cuh>
 
-static void bench_sort(nvbench::state& state)
+template <typename DataType>
+static void bench_sort(nvbench::state& state, nvbench::type_list<DataType>)
 {
-  auto const stable = static_cast<bool>(state.get_int64("stable"));
-  auto const n_rows = static_cast<cudf::size_type>(state.get_int64("n_rows"));
-  auto const n_cols = static_cast<cudf::size_type>(state.get_int64("n_cols"));
-  auto const nulls  = state.get_float64("nulls");
+  auto const stable    = static_cast<bool>(state.get_int64("stable"));
+  auto const n_rows    = static_cast<cudf::size_type>(state.get_int64("num_rows"));
+  auto const n_cols    = static_cast<cudf::size_type>(state.get_int64("num_cols"));
+  auto const nulls     = state.get_float64("nulls");
+  auto const data_type = cudf::type_to_id<DataType>();
 
-  // Create table with values in the range [0,100)
   data_profile const profile =
     data_profile_builder().cardinality(0).null_probability(nulls).distribution(
```

Comment:
```suggestion
    data_profile_builder().cardinality(0).null_probability(null_probability).distribution(
```

Reply:
I'd rather keep the `nulls` name since it matches the parameter name which will be the column name in the output which I'd like to keep small-ish.
I agree with the num_rows, num_cols since it matches this rule and these names are consistent without nvbench benchmarks.